src/c++/perf_analyzer/test_perf_utils.cc

```diff
@@ -370,5 +370,56 @@ TEST_CASE("perf_utils: TensorToRegionName")
   CHECK(TensorToRegionName("") == "");
 }
 
+TEST_CASE("perf_utils: Test string input numbers")
+{
+  std::string pos{"123"};
+  std::string neg{"-123"};
+  std::string zero{"0"};
+  std::string not_integer{"helloworld"};
+  std::string almost_integer{"123hello"};
+  std::string empty{""};
+
+  SUBCASE("IsPositiveInteger")
+  {
+    CHECK(IsPositiveInteger(pos));
```

Comment:
nit: can you rewrite these so that it's `IsPositiveInteger(...) == true` and `IsPositiveInteger(...) == false` so it's more readable?